packages/tokamak/contracts/hardhat.config.ts

```diff
@@ -20,23 +23,25 @@ import './tasks'
 // Load environment variables from .env
 dotenv.config()
 
-const { TASK_COMPILE_SOLIDITY_GET_SOLC_BUILD } = require("hardhat/builtin-tasks/task-names");
```

Comment:
Why we need it?

Reply:
There is an issue on ARM architecture. When we build the repository, it can't download only specific solidity version(`0.5.17+commit.d19bba13`). I don't know why it occurs exactly. And @nguyenzung resolved this issue adding WASM of it manually.